### ISSUE TITLE
Launchpad Pro Polyphonic Aftertouch Issues

### ISSUE THREAD

**Issue by TheYokai** [git-moss/DrivenByMoss#110]
Hi there, I love the work you are doing with these scripts. However, I have a question regarding the Launchpad Pro and Polyphonic Aftertouch. I am currently unable to get any consistent polyphonic aftertouch readings when trying to use the pressure modulation within the expressions modulator. 

Changing the setting inside the control script to be channel-based aftertouch works as expected, but it would be nice to get some per note aftertouch as well. Sorry, I'm new to bitwig so it could be very possible that I'm just doing something wrong here. 

Here are my system details:
```
 OS: Ubuntu 18.10 cosmic
 Kernel: x86_64 Linux 4.18.0-13-generic
 DE: GNOME 
 WM: GNOME Shell
 CPU: Intel Core i7-4770K @ 8x 3.9GHz [27.8°C]
 GPU: GeForce GTX 760
 RAM: 11945MiB
```

Basically, the problem is that aftertouch seems to be looking at pressure with the wrong notes. For example, say I'm pressing pad `[0,0]` -- the only way to get pressure readings is to apply pressure on the pad `[0,3]` or `[5,2]` That same pattern can be shifted over until `[2,0]`. Similarly, pad `[4,4]` responds to pressure at pad `[6,5]`. Lastly, pad `[0,4]` responds to pressure on the note an octave higher. I've tried this on multiple key offsets and scales and it doesn't change anything. If I were to take a guess, there is some index math that is incorrect which is causing the pad pressure assignments to be inconsistent.

**Comment by git-moss** [comment]
Sorry, but I cannot reproduce that. I tried several scales. How are you monitoring those values?

**Comment by TheYokai** [comment]
Ok I'll look around for a linux equivalent. If I were to guess, there should be a jack equivalent.

**Comment by TheYokai** [comment]
So I've been trying to look into this with no luck. I've tried this on both windows and linux and both of them have this strange pressure mapping problem. Also, I've tried a few midi previewing programs (midi-ox on windows) and for some reason it is having a hard time actually recognizing midi commands from my launchpad. I was also having problems with some jack-based linux midi previewing programs, but wrote it off at the time as probably just my system configuration.

**Comment by git-moss** [comment]
Sounds like you either have a faulty device or there is some misconfiguration, check if you can reset it.

**Comment by TheYokai** [comment]
I RTFM'd more, and it was a configuration issue. Make sure you have polyphonic aftertouch enabled in the `setup` page in order for this to work. 

Leaving this comment here in case someone else has this problem and hasn't read the manual (like you should always do.)

Thanks for taking the time to respond to this issue request @git-moss .

**Comment by von-Elsewhere** [comment]
So do you still remember how to access the setup mode @Eoin-ONeill-Yokai? I haven't found Launchpad Pro Mk1 manual online, nor do I have one, for reference.
